pantheon.php

```diff
@@ -45,6 +42,9 @@
     if ( getenv( 'FRAMEWORK' ) === 'wordpress_network' && ! defined( 'WP_ALLOW_MULTISITE' ) ) {
         define( 'WP_ALLOW_MULTISITE', true );
     }
+    if ( defined( 'MULTISITE' ) && defined( 'WP_ALLOW_MULTISITE' ) && WP_ALLOW_MULTISITE ) {
+	require_once 'inc/pantheon-network-setup.php';
+    }
```

Comment:
These lines are unrelated and merged in #21. Can you remove these from the PR?

Reply:
Thanks for the review. merged with the latest branch